{"uid":"3e86da95be98e32e","name":"test_log_rotate","fullName":"tests.integration.test_charm#test_log_rotate","historyId":"5928f982f61b2ffbcbdb89ed640ec1aa","time":{"start":1729043308858,"stop":1729043308858,"duration":0},"description":"Test that log are being rotated.","descriptionHtml":"<p>Test that log are being rotated.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1729042691039,"stop":1729042691039,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1729042691040,"stop":1729042691040,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1729042691040,"stop":1729042691203,"duration":163},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1729043308858,"stop":1729043308858,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/ubuntu/actions-runner/_work/mongodb-operator/mongodb-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1729043308862,"stop":1729043309144,"duration":282},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1729043309145,"stop":1729043309145,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1729043309145,"stop":1729043309145,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1729043309145,"stop":1729043309146,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_charm"},{"name":"host","value":"github-runner"},{"name":"thread","value":"20200-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"2f7dfc3fc6186d7c","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":7,"passed":70,"unknown":2,"total":79},"items":[{"uid":"87388953501eb81a","reportUrl":"../2352//#testresult/87388953501eb81a","status":"passed","time":{"start":1728959432531,"stop":1728960098009,"duration":665478}},{"uid":"d1903a61b9f4301f","reportUrl":"../2349//#testresult/d1903a61b9f4301f","status":"passed","time":{"start":1728874014624,"stop":1728874680286,"duration":665662}},{"uid":"16fc6a031de08bf4","reportUrl":"../2348//#testresult/16fc6a031de08bf4","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728788628428,"stop":1728788628428,"duration":0}},{"uid":"1f5edadefa26975a","reportUrl":"../2347//#testresult/1f5edadefa26975a","status":"passed","time":{"start":1728701689193,"stop":1728702354574,"duration":665381}},{"uid":"f3a44096541acbea","reportUrl":"../2346//#testresult/f3a44096541acbea","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1728615326345,"stop":1728615326345,"duration":0}},{"uid":"d0bc6518f5ed927a","reportUrl":"../2342//#testresult/d0bc6518f5ed927a","status":"passed","time":{"start":1728529154976,"stop":1728529820496,"duration":665520}},{"uid":"f912c9e97496df67","reportUrl":"../2340//#testresult/f912c9e97496df67","status":"passed","time":{"start":1728442014750,"stop":1728442680416,"duration":665666}},{"uid":"b6a8cc940eafadb4","reportUrl":"../2338//#testresult/b6a8cc940eafadb4","status":"passed","time":{"start":1728354495215,"stop":1728355161834,"duration":666619}},{"uid":"c460012099095f1","reportUrl":"../2335//#testresult/c460012099095f1","status":"passed","time":{"start":1728268775711,"stop":1728269441235,"duration":665524}},{"uid":"ce892bc3a960fd7b","reportUrl":"../2334//#testresult/ce892bc3a960fd7b","status":"passed","time":{"start":1728181939090,"stop":1728182604801,"duration":665711}},{"uid":"49e6ec6bf8934c7e","reportUrl":"../2333//#testresult/49e6ec6bf8934c7e","status":"passed","time":{"start":1728096590374,"stop":1728097256033,"duration":665659}},{"uid":"262c245b08b4b604","reportUrl":"../2330//#testresult/262c245b08b4b604","status":"passed","time":{"start":1728009352265,"stop":1728010017774,"duration":665509}},{"uid":"17b800cd7b5c79dd","reportUrl":"../2320//#testresult/17b800cd7b5c79dd","status":"passed","time":{"start":1727921857918,"stop":1727922523433,"duration":665515}},{"uid":"4854246ecc434df7","reportUrl":"../2310//#testresult/4854246ecc434df7","status":"passed","time":{"start":1727837531482,"stop":1727838197145,"duration":665663}},{"uid":"d71179c916be1355","reportUrl":"../2307//#testresult/d71179c916be1355","status":"passed","time":{"start":1727750135271,"stop":1727750800752,"duration":665481}},{"uid":"fcf2c3fca08432f2","reportUrl":"../2304//#testresult/fcf2c3fca08432f2","status":"passed","time":{"start":1727665086856,"stop":1727665752293,"duration":665437}},{"uid":"e758237b67177cde","reportUrl":"../2303//#testresult/e758237b67177cde","status":"passed","time":{"start":1727576412260,"stop":1727577077981,"duration":665721}},{"uid":"36a144fb4adf98ad","reportUrl":"../2302//#testresult/36a144fb4adf98ad","status":"passed","time":{"start":1727490576884,"stop":1727491243570,"duration":666686}},{"uid":"854e4d68a790c6b7","reportUrl":"../2300//#testresult/854e4d68a790c6b7","status":"passed","time":{"start":1727405787264,"stop":1727406455045,"duration":667781}},{"uid":"4cca4b1f2a5a155d","reportUrl":"../2299//#testresult/4cca4b1f2a5a155d","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727320015414,"stop":1727320015414,"duration":0}}]},"tags":["runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])","asyncio","group(1)"]},"source":"3e86da95be98e32e.json","parameterValues":[]}